lib/E2E/TensorToMemref/LowerShapedResultsToMemref.cpp

```diff
@@ -242,7 +242,11 @@ namespace {
 // TODO: Don't hardcode the lowering for every op in this one pass.
 class LowerShapedResultsToMemref
     : public LowerShapedResultsToMemrefBase<LowerShapedResultsToMemref> {
-  void runOnOperation() {
+  void getDependentDialects(DialectRegistry &registry) const override {
+    registry.insert<linalg::LinalgDialect, scf::SCFDialect>();
```

Comment:
can you add shape::ShapeDialect here? I think that should fix the test.

Reply:
Just noticed my mistake and fixed it.